test/src/Provider/DocusignTest.php

```diff
@@ -60,7 +60,7 @@ public function getUrlDataProvider()
         ];
     }
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
Adding these return types in means that we'll have to drop support for PHP 5.6 and 7.0. But I think it's required in PHPUnit 8+ right? I think it's fine to stop supporting unsupported PHP versions though?